l10n-pocket/configs/vendor.toml

```diff
@@ -0,0 +1,26 @@
+# All content handled by the vendor or Mozilla staff should
+# be covered by the [[paths]] in this file, with the relevant locales.
+
+basepath = ".."
+
+locales = [
+    "de",
+    "en",
+    "es-ES",
```

Comment:
TODO: add `es` here as well, I think (which will be mapped to `es-AR`, `es-CL`, `es-MX`